config/core/configmaps/observability.yaml

```diff
@@ -95,11 +95,25 @@ data:
     # It supports either prometheus (the default) or opencensus.
     metrics.backend-destination: prometheus
 
+    # metrics.reporting-period-seconds specifies the global metrics reporting period for control and data plane components.
+    # If a zero or negative value is passed the default reporting period is used (10 secs).
+    # If the attribute is not specified a default value is used per metrics backend.
+    # For the prometheus backend the default reporting period is 5s while for opencensus it is 60s.
+    metrics.reporting-period-seconds: "10"
+
     # metrics.request-metrics-backend-destination specifies the request metrics
     # destination. It enables queue proxy to send request metrics.
     # Currently supported values: prometheus (the default), opencensus.
     metrics.request-metrics-backend-destination: prometheus
 
+    # metrics.request-metrics-reporting-period-seconds specifies the request metrics reporting period in sec at queue proxy.
+    # If a zero or negative value is passed the default reporting period is used (10 secs).
+    # If the attribute is not specified, it is overriden by the value of metrics.reporting-period-seconds.
+    metrics.request-metrics-reporting-period-seconds: "10"
+
+    # metrics.opencensus-address specifies the metrics collector address. Metrics are shiped using the opencensus format.
+    metrics.opencensus-address: "otel-collector.default.svc:55678"
```

Comment:
This leaks the implementation detail (opencensus). It is better to have it as a reminder to change the name later on than hiding this option from the end user.

Reply:
Will add it in another PR.